nih-sparc/sparc-app
# Resources landing page and detail page #4c6eyf

# Description
The purpose of this PR is to add the resources landing page and detail pages. Note: Designs for these pages are not done, so this is to test functionality and "basic page layout" only.

## Type of change


# How Has This Been Tested?
- Open the Resources page from the main navigation
- You should see resources listed
- Click on one of the resource's name
- You should be taken to that resource's detail page.

# Checklist:
